package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A GraphQL MongoDB server.",
   "main": "index.js",
   "scripts": {
-    "start": "node server.js",
+    "start": "npm run server",
```

Comment:
that was a small bug... it was saying cannot find server.js (looking for the non existed file).
now it should run npm server :)